app/forms/collection_form.rb

```diff
@@ -26,7 +26,8 @@ def sync(*)
     super
   end
 
-  validates :name, :description, :contact_email, :managers, :access, presence: true
+  validates :name, :description, :managers, :access, presence: true
+  validates :contact_email, presence: true, format: { with: Devise.email_regexp }
```

Comment:
Can/should we remove the corresponding model validation now that we have the form validation? Cc: @jcoyne

Reply:
I don't care. This seems reasonable to have in both places.